docs/.vuepress/theme/layouts/GlobalLayout.vue

```diff
@@ -51,7 +51,7 @@
             </div>
             <div>
               <a
-                href="https://filecoinproject.slack.com"
+                href="/community/contact-us/"
```

Comment:
```suggestion
                href="/community/contact-us#questions-and-community-discussion"
```
Direct link to the relevant header

Reply:
@jessicaschilling I did try the layout with headers but didn't feel there was enough content to justify it. I'll try setting it up your suggested way with the new content and see if it feels less silly than when I did it. 😂
![image](https://user-images.githubusercontent.com/19171465/126552337-7d9c4bb9-d0ee-46ba-a803-8f50e2b41682.png)

Will circle back and fix the link since the `contact-us` portion won't be correct but I need to play with what the asterisk causes to happen.